consensus/module.go

```diff
@@ -216,6 +188,24 @@ func (m *ConsensusModule) SetBus(pocketBus modules.Bus) {
 	m.leaderElectionMod.SetBus(pocketBus)
 }
 
+func (*ConsensusModule) ValidateConfig(cfg modules.Config) error {
+	if _, ok := cfg.Consensus.(*typesCons.ConsensusConfig); !ok {
+		return fmt.Errorf("cannot cast to ConsensusConfig")
+	}
+	return nil
+}
+
+func (*ConsensusModule) ValidateGenesis(genesis modules.GenesisState) error {
+	if _, ok := genesis.ConsensusGenesisState.(*typesCons.ConsensusGenesisState); !ok {
+		return fmt.Errorf("cannot cast to ConsensusGenesisState")
+	}
+	return nil
+}
+
+func (*ConsensusModule) GetPrivateKey(runtime modules.Runtime) (cryptoPocket.PrivateKey, error) {
```

Comment:
Do we not need the `sholdUseRandomPk` part here?

Reply:
not anymore

![image](https://user-images.githubusercontent.com/29378614/192514698-6deed7eb-d88d-4121-a9ac-bf7d1e3bba27.png)


😉

Reply:
Nice!